lib/dao.js

```diff
@@ -1956,8 +2016,16 @@ DataAccessObject.updateAll = function (where, data, options, cb) {
 
     function updateCallback(err, info) {
       if (err) return cb (err);
+
       var context = {
         Model: Model,
+        data: data,
+        hookState: hookState,
+        options: options
+      };
+
+      context = {
+        Model: Model,
```

Comment:
This looks like a forgotten code from "loaded" hook that was not removed completely. Could you please remove/revert? I think there should be no changes in the `updateAll` method.